app/lib/database.py

```diff
@@ -49,99 +48,131 @@ def __init__(self, cursor):
         assert(cursor is not None)
         self._cursor = cursor
 
-    def create_json_table(self, table_name: str):
+    def create_metadata_file_view(self, table_name: str, file_type: str):
         query = self._prepare_statement(
             """
-            CREATE TABLE IF NOT EXISTS {} (
-                uuid UUID PRIMARY KEY NOT NULL,
-                json JSONB NOT NULL
-            );
+            CREATE OR REPLACE VIEW {} AS
+            SELECT * FROM metadata_files
+            WHERE metadata_files.file_type = %s
```

Comment:
What do you think about creating a file_type enum table and adding a row to it for every new file_type